pkg/handler/handler_test.go

```diff
@@ -121,14 +78,49 @@ func TestReadmeUtgaveResolvers(t *testing.T) {
 	if res := r.UTGAVE(); res != r.Utgave {
 		t.Errorf("Expected utgave %d, got %d", res, r.Utgave)
 	}
-	if res := r.IMAGE(); res != "https://readme.abakus.no/test.png" {
+	if res := r.IMAGE(); res != "test.png" {
 		t.Errorf("Expected image %s, got %s", res, r.Image)
 	}
-	if res := r.PDF(); res != "https://readme.abakus.no/2.pdf" {
+	if res := r.PDF(); res != "2.pdf" {
 		t.Errorf("Expected pdf %s, got %s", res, r.Pdf)
 	}
 }
 
+func TestLatestReadmeOnline(t *testing.T){
+	if offline == nil || *offline {
+		t.Skip()
+	}
+
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx, option.WithoutAuthentication())
+	if err != nil {
+		t.Errorf("Error while creating storage client: %+v", err.Error())
+	}
+
+	r := resolver{
+		*client,
+		ctx,
+	}
+
+	first, err := r.LatestReadme()
+	if err != nil {
+		t.Errorf("Got error %e\n", err)
+	}
+	if first == nil {
+		t.Error("Expected to get a first readme, but got nil.")
+	}
+}
+
+// We create a new function that can take test data so that test data is consistent
+func LatestReadme(readmes []ReadmeUtgave) (*ReadmeUtgave, error) {
```

Comment:
Is there a reason to export this?

Also https://blog.golang.org/godoc

Reply:
Nope.